Index: src/LearnEnglish/LearnJFrame.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package LearnEnglish;\r\n\r\nimport javax.swing.*;\r\n\r\npublic class LearnJFrame extends JFrame {\r\n    public LearnJFrame() {\r\n        //设置宽高\r\n        this.setSize(400,400);\r\n        //设置标题\r\n        this.setTitle(\"英语救星 学习 v1.0\");\r\n        //设置居中\r\n        this.setLocationRelativeTo(null);\r\n        //设置始终位于最上层\r\n        this.setAlwaysOnTop(true);\r\n        //设置关闭模式\r\n        this.setDefaultCloseOperation(3);\r\n\r\n        //设置是否显示\r\n        this.setVisible(true);\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/LearnEnglish/LearnJFrame.java b/src/LearnEnglish/LearnJFrame.java
--- a/src/LearnEnglish/LearnJFrame.java	(revision 549ddf382e5868d9627244190418e650ff91b00d)
+++ b/src/LearnEnglish/LearnJFrame.java	(date 1716550884594)
@@ -14,6 +14,8 @@
         this.setAlwaysOnTop(true);
         //设置关闭模式
         this.setDefaultCloseOperation(3);
+        //设置菜单界面
+
 
         //设置是否显示
         this.setVisible(true);
Index: src/A20240525/AboutString.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package A20240525;\r\n\r\npublic class AboutString {\r\n    public static void main(String[] args) {\r\n        String s1 = new String(\"123\");\r\n        String s2 = new String(\"123\");\r\n        System.out.println(s1 == s2);\r\n        System.out.println(s1.equals(s2));\r\n\r\n        System.out.println(\"=== === ===\");\r\n\r\n\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/A20240525/AboutString.java b/src/A20240525/AboutString.java
--- a/src/A20240525/AboutString.java	(revision 549ddf382e5868d9627244190418e650ff91b00d)
+++ b/src/A20240525/AboutString.java	(date 1716605864770)
@@ -9,6 +9,8 @@
 
         System.out.println("=== === ===");
 
-
+        String s3 = "123";
+        String s4 = "123";
+        System.out.println(s3 == s4);
     }
 }
