Index: src/A20240802/note/README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## 线程和进程\r\n\r\n### 基本资料\r\n* https://www.ruanyifeng.com/blog/2013/04/processes_and_threads.html\r\n* https://docs.oracle.com/javase/tutorial/essential/concurrency/procthread.html\r\n\r\n## 继承并使用 Thread 类，使用它的 run() 方法和 start() 方法\r\n[ThreadTest1.java](..%2FThreadTest1.java)\r\n![img.png](img.png)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/A20240802/note/README.md b/src/A20240802/note/README.md
--- a/src/A20240802/note/README.md	(revision bb6cc2ee6d306f6d09226a8504bd59ae3324a6b2)
+++ b/src/A20240802/note/README.md	(date 1722582521609)
@@ -6,4 +6,14 @@
 
 ## 继承并使用 Thread 类，使用它的 run() 方法和 start() 方法
 [ThreadTest1.java](..%2FThreadTest1.java)
-![img.png](img.png)
\ No newline at end of file
+![img.png](img.png)
+
+## Thread 的静态方法
+
+### sleep
+
+### yield
+
+### setPriority
+
+### isAlive
\ No newline at end of file
